crates/primitives/src/transaction/mod.rs

```diff
@@ -1126,6 +1128,11 @@ impl TransactionSigned {
         &self.signature
     }
 
+    /// Transaction
+    pub const fn transaction(&self) -> &Transaction {
+        &self.transaction
+    }
```

Comment:
Assume this is okay as we already have one for hash and signature. Need it below in the `impl SignedTransaction`